VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs

```diff
@@ -16,6 +17,8 @@ public interface IEventHubClient
         Task AdminConsultationMessage(Guid conferenceId, RoomType room, string requestedFor,
             ConsultationAnswer? answer = null);
         Task HelpMessage(Guid conferenceId, string participantName);
+        
+        Task ReceiveMessage(Guid conferenceId, string from, string message, DateTime timestamp);
```

Comment:
Rename to ReceiveMessageAsync

Reply:
Usually, yes but I do not think it's applicable in this context. The Task return type is a requirement of an hub message signature but clients would subscribe to the message name. Async wouldn't be necessary here see [chat-hub.service.ts "getChatMessage()"](https://github.com/hmcts/vh-video-web/pull/375/files#diff-51e500d4afbc36df6123ba5937926fab)